src/actions/CommentActions/CommentGetAction.js

```diff
@@ -0,0 +1,89 @@
+import actionTypes from "../actionTypes";
+
+const token = window.localStorage.getItem("token");
+
+export const ReplySuccess = message => ({
+  type: actionTypes.REPLYSUCCESS,
+  payload: message
+});
+
+export const GETCOMMENT = comments => ({
+  type: actionTypes.GETCOMMENT,
+  payload: comments
+});
+
+export const ReplyPostAction = (id, data, slug) => dispatch => {
+  return fetch(
+    `${process.env.REACT_APP_API_URL_BASE}articles/${slug}/comments/${id}/`,
+    {
+      method: "POST",
+      headers: {
+        "content-type": "application/json",
+        Authorization: `Bearer ${token}`
+      },
+      CORS: "no-cors",
+      body: JSON.stringify({ reply: data })
+    }
+  )
+    .then(res => res.json())
+    .then(response => {
+      dispatch(ReplySuccess(response));
+    });
+};
+
+export const CommentGetAction = (data, slug) => dispatch => {
+  return fetch(
+    `${process.env.REACT_APP_API_URL_BASE}articles/${slug}/comments/${data}`,
+    {
+      method: "GET",
+      headers: {
+        "content-type": "application/json",
+        Authorization: `Bearer ${token}`
+      },
+      CORS: "no-cors"
+    }
+  )
+    .then(res => res.json())
+    .then(response => {
+      if (response.comments) {
```

Comment:
@Kasulejoseph This code is verbose and not easy to follow. Look into functional programming [concepts](https://codeburst.io/functional-programming-in-javascript-e57e7e28c0e5) to write more readable code. This can also be refactored into a function.

Reply:
Thanks for the resource, let me get the idea and implement it.

Reply:
This might take longer to implement. Just consider it in future implementations